pywbem/_server.py

```diff
@@ -735,31 +735,34 @@ def get_central_instances(self, profile_path, central_class=None,
         * Central class methodology
         * Scoping class methodology
 
-        Use of the scoping class methodology requires specifying the central
-        class, scoping class and scoping path defined by the profile. If any
-        of them is `None`, this method will attempt only the
-        GetCentralInstances and central class methodologies, but not the
-        scoping class methodology.
-        If using these two methodologies does not result in any central
-        instances, and the scoping class methodology cannot be used,
-        :exc:`~pywbem.ModelError` is raised.
-
-        The scoping path is a directed traversal path from the central
-        instances to the scoping instances. Its first list item is always
-        the association class name of the traversal hop starting at the
-        central instances. For each further traversal hop, the list contains
-        two more items: The class name of the near end of that hop, and
-        the class name of the traversed association.
-        As a result, the class names of the central instances and scoping
-        instances are not part of the list.
-
-        Example for a 1-hop traversal:
+        All three methodologies start from the CIM_RegisteredProfile instance
+        referenced by the `profile_path` parameter.
+
+        Autonomous profiles implement the GetCentralInstances methodology
+        and/or the central class methodology.
+        Component profiles implement any one or more of these methodologies.
+
+        The scoping class methodology requires specifying the central class,
+        scoping class and scoping path defined by the management profile;
+        the other methodologies do not require these parameters.
+
+        :term:`DSP1001` requires that all management profiles specify a
+        central class. In addition, component profiles are required to specify
+        scoping class and scoping path. The scoping path is a traversal path
+        from the central class to the scoping class.
+
+        Older component profiles may not follow the requirement to specify
+        scoping class and scoping path. In that case, the scoping class and
+        scoping path can often be determined from the class diagram. Most of
```

Comment:
Should probably say class diagram in the specification for the profile.

Reply:
DONE